docker-compose/1_Auto_Upstall/noco.sh

```diff
@@ -510,6 +514,9 @@ create_docker_compose_file() {
     if [ "${CONFIG_MINIO_ENABLED}" = "Y" ]; then
         gen_minio=1
     fi
+    if [ "${CONFIG_MINIO_ENABLED}" = "Y" ]; then
```

Comment:
Should be `["${CONFIG_POSTGRE_SQLITE}" = "p"] || ["${CONFIG_POSTGRE_SQLITE}" = "P" ]`